2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_setup.py:_flush():69] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2021-10-21 00:03:41,835 WARNING MainThread:53659 [wandb_setup.py:_flush():69] could not save program above cwd: /Users/obara/miniforge3/envs/mlflow-827614215fe534cb79b9b720eb621b7f7070f8a8/bin/pytest
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20211021_000341-1ibftf0q/logs/debug.log
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20211021_000341-1ibftf0q/logs/debug-internal.log
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_init.py:init():375] calling init triggers
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-21 00:03:41,835 INFO    MainThread:53659 [wandb_init.py:init():424] starting backend
2021-10-21 00:03:41,835 INFO    MainThread:53659 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-10-21 00:03:41,856 INFO    MainThread:53659 [backend.py:ensure_launched():135] starting backend process...
2021-10-21 00:03:41,859 INFO    MainThread:53659 [backend.py:ensure_launched():139] started backend process with pid: 53775
2021-10-21 00:03:41,860 INFO    MainThread:53659 [wandb_init.py:init():429] backend started and connected
2021-10-21 00:03:41,861 INFO    MainThread:53659 [wandb_init.py:init():477] updated telemetry
2021-10-21 00:03:41,861 INFO    MainThread:53659 [wandb_init.py:init():500] communicating current version
2021-10-21 00:03:42,238 INFO    MainThread:53659 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-21 00:03:42,238 INFO    MainThread:53659 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-21 00:03:42,460 INFO    MainThread:53659 [wandb_init.py:init():540] starting run threads in backend
2021-10-21 00:03:42,909 INFO    MainThread:53659 [wandb_run.py:_console_start():1601] atexit reg
2021-10-21 00:03:42,910 INFO    MainThread:53659 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-10-21 00:03:42,910 INFO    MainThread:53659 [wandb_run.py:_redirect():1480] Redirecting console.
2021-10-21 00:03:42,911 INFO    MainThread:53659 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-21 00:03:42,911 INFO    MainThread:53659 [wandb_init.py:init():565] run started, returning control to user process
2021-10-21 00:03:49,504 INFO    MainThread:53659 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-21 00:03:49,504 INFO    MainThread:53659 [wandb_run.py:_restore():1543] restore
2021-10-21 00:03:52,422 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:03:53,365 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:03:53,569 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 1047
}

2021-10-21 00:03:54,523 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 4456
}

2021-10-21 00:03:54,629 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 7067
}

2021-10-21 00:03:54,757 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1047
  total_bytes: 7067
}

2021-10-21 00:03:54,863 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:54,969 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:55,075 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:55,181 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:55,285 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:55,484 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:55,590 INFO    MainThread:53659 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7067
  total_bytes: 7067
}

2021-10-21 00:03:56,665 INFO    MainThread:53659 [wandb_run.py:_show_files():1915] logging synced files
